### ISSUE TITLE
Ideas

### ISSUE THREAD

**Issue by LeStahL** [LeStahL/spacerace69#1]

**Comment by LeStahL** [comment]
Szenenideen: 
- Semyorka r7 start (sputnik rakete). Zeige beschriftung auf russisch, sowie authentischen start.
- Ariane 6 start. Zeige beschriftun auf englisch
- View aus dem Cockpit mit jeweils hund, nilpferd und dino; zeige fancy ui

**Comment by LeStahL** [comment]
Szene 1: Start mit countdown, ist von raketenoberfläche aus zu sehen, Boden fliegt weg
Szenenidee: Wegweiser in die eine richtung: Compos, in die andere Richtung: Uranus, einmal venus (danach tittenmaus szene)
Schwarzes Loch + Wurmloch  ins Wurmloch der Coverlady - Tunnelszene

**Comment by LeStahL** [comment]
Viel mit so Sternen, Szenen um planeten / swingby manöver
asteroidenfeld mit raketen auf steine feuern

**Comment by LeStahL** [comment]
Szene wo man auf nem planet steht wo ein dino zu sehen ist / urerde / geile landschaft, wo die raketen oben drüber fliegen und zu sehen sind, versehentlich wird der planet dann aber mit roten knöpfen in die luft gejagt

**Comment by LeStahL** [comment]
fliegen teilweise auf spaceautobahn wie bei supermario, dann einer schmeisst pilze, noch mal researchen / immi fragen was für mario witze da gehen

**Comment by LeStahL** [comment]
eine rakete wird irgendwann getroffen und verliert teile, auf denen stehen dann die greetings als hersteller logo

**Comment by LeStahL** [comment]
zeige schwein mit raumanzug das irgendwo fliegt

**Comment by LeStahL** [comment]
geile landschaften: gaswolken, planeten, galaxien, sterne, nebel,

**Comment by LeStahL** [comment]
ne gravitationswelle zeigen, doppelsternsystem mit akkretionsscheibe

**Comment by LeStahL** [comment]
zeige starwars referenzen / todesstern

**Comment by LeStahL** [comment]
gravitationslinseneffekt-postshader
rotverschiebungs-postshader

**Comment by qm210** [comment]
https://www.starobserver.org/image/1311/bhbinary_xmm_1728.jpg
https://www.tat.physik.uni-tuebingen.de/~tat/blog/content/public/upload/1*vr4ryfxua9fbvmkszafjiq_2_o.jpg

**Comment by LeStahL** [comment]
raketen fliegen durch die nebel und verursachen turbulenzen
text: "wait, what!?"

**Comment by qm210** [comment]
http://scienceblogs.de/astrodicticum-simplex/wp-content/blogs.dir/28/files/2012/07/i-10d91d0b2caa3b05ebe17d206d907cfb-Eagle_nebula_pillars.jpg
adlernebel / pillars of creation

**Comment by qm210** [comment]
OH SHIT ITS ANOTHER SUPERNOVA

**Comment by LeStahL** [comment]
Ständig schnelle supernova explosionen - "oh shit, another supernova!"

**Comment by qm210** [comment]
http://cdn.eso.org/images/screen/eso9948f.jpg

**Comment by qm210** [comment]
http://www.astroimages.de/pics/gallery/full/NGC2244-20100307.jpg

**Comment by qm210** [comment]
YEAH, sowas und dann fliegen die drei viecher oben am himmel so als kometen vorbei https://upload.wikimedia.org/wikipedia/commons/thumb/c/c9/Perseiid.jpg/1920px-Perseiid.jp

**Comment by LeStahL** [comment]
ewok starwars anspielung

**Comment by LeStahL** [comment]
quake 4 symbole als logo für "evoke"

**Comment by qm210** [comment]
greetings: dummer text / demogruppen-namen so star wars style nach hinten fliegen lassen

"evoke" schriftzug in quake-schrift / klingonisch / whatever

**Comment by LeStahL** [comment]
schalter speed hebel blinkende alarme cockpit mit lächerlich vielen controls

**Comment by LeStahL** [comment]
letzte szene: crash aufn planet, einer knallt den anderen mit lächerlich großer waffe ab. danach "THE END."

**Comment by zimmmx** [comment]
https://youtu.be/x9xFRch7N5o?t=80
Eigentlich der gesamte clip des Raketenstarts.
Besonders Kameraeinstellung in der Kamera an Rakete zur Erde schaut (t=91).
Da vielleicht die Saturn V/SLS gegen Sojuz austauschen.

**Comment by zimmmx** [comment]
https://www.youtube.com/watch?v=u_zHHwH2JSQ
Lohnt am Schluss. Eventuell das kopieren?

**Comment by LeStahL** [comment]
dickbutt und kaktus der sabbert ins cockpit
